.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
@vickywane same point here, LH is not intended for serverless function testing.

Reply:
I'll remove it